modules/pixfutureBidAdapter.md

```diff
@@ -1,5 +1,10 @@
+---
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+---
```

Comment:
These adapter documentation highlights need to go in our docs repo here instead please -> https://github.com/prebid/prebid.github.io

Reply:
We have located this information in the docs repo and removed it from the current .md file.